database/factories/ArticleFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Article;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class ArticleFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Article::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'slug' => $this->faker->unique()->word,
+            'title' => $this->faker->title,
+            'body' => $this->faker->sentence,
+            'owner_id' => '1'
```

Comment:
Здесь нужно применить фабрику пользователей. Вряд ли для фабрик удобно, что все статьи привязываются к админу
```suggestion
     'owner_id' => User::factory(),
```